packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -108,7 +108,7 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
     <Stack className={mergedRootStyle}>
       <div className={mergedTextContainerStyle}>
         <TextField
-          autoFocus={props.autoFocus === 'sendBoxTextField'}
+          autoFocus={false}
```

Comment:
why have we removed this?

Reply:
Removed it temporarily as it was highlighting the text filed and now we need focus on the outer div of sandbox.

Reply:
I assume we are planning to put this back?